torch/csrc/jit/python/script_init.cpp

```diff
@@ -1693,6 +1693,15 @@ void initJitScriptBindings(PyObject* module) {
         std::istringstream in(buffer);
         return _get_model_bytecode_version(in);
       });
+  py::class_<OperatorInfo>(m, "OperatorInfo")
+      .def_readonly("num_schema_args", &OperatorInfo::num_schema_args);
+  m.def("_get_model_ops_and_info", [](const std::string& filename) {
+    return _get_model_ops_and_info(filename);
+  });
+  m.def("_get_model_ops_and_info", [](const std::string& buffer) {
+    std::istringstream in(buffer);
+    return _get_model_ops_and_info(in);
+  });
```

Comment:
The input type is the same. I'm not sure if the binding between c++ and python can work well here. Recommend to rename the one read from buffer `_get_model_ops_and_info` to `_get_model_ops_and_info_from_buffer`.

Reply:
oh good catch I meant to name it from_buffer